mediators/datamapper/components/org.wso2.carbon.gateway.mediators.datamapper.engine/src/main/java/org/wso2/carbon/gateway/mediators/datamapper/engine/core/exceptions/InvalidPayloadException.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.gateway.mediators.datamapper.engine.core.exceptions;
+
+/**
+ * This exception is thrown when engine gets a invalid payload with respect to schema
+ */
+public class InvalidPayloadException extends Exception {
+
+    private String message = null;
+
+    public InvalidPayloadException(String message) {
+        super(message);
+        this.message = message;
+    }
+
+    @Override public String toString() {
```

Comment:
can't we set @Override annotation on top of method signature instead of mentioning it in s single line?(Most of the places I saw are like that, IMO better we can have a consistent way, either in single line or before method signature)